### DISCUSSION TITLE
debug 'Apple Music is not available in your country or region'

### DISCUSSION THREAD

**Discussion by dabl2** [ciderapp/Apple-Music-Electron#137] [Q&A]
[config.zip](https://github.com/cryptofyre/Apple-Music-Electron/files/6823603/config.zip)
[main.log](https://github.com/cryptofyre/Apple-Music-Electron/files/6823606/main.log)
I just installed v2.3.0 on my Windows 10 machine for the first time.

I'm getting 'Apple Music is not available in your country or region' on login attempt.

I'm located in the USA

I've read through the [release notes](https://github.com/cryptofyre/Apple-Music-Electron/releases/tag/v2.3.0) and found 'config.json' in my Documents folder but don't see anything in that file referencing web security. 

I also saw the note about 'authMode' on the [Troubleshooting](https://github.com/cryptofyre/Apple-Music-Electron/wiki/Troubleshooting) page but again don't see anything in config.json referencing that nor do I see a checkbox in the UI for it.

Is config.json where I want to be looking for either the web security or authMode settings?

I see "allowRunningInsecureContent" in config.json and tried changing that from true to false and restarted the app but don't see a change in login behavior.

I tried adding enableWebSecurity: true, to config.json and restarted the app but again don't see any change in login behavior

I'm not seeing anything obvious in %APPDATA%\Apple Music\logs\main.log other than confirmation those changes appeared to be getting processed and that cfg matches the config.json in my Documents folder etc.

The attached config.zip contains the default/out of the box config.json file for reference.

EDIT: I guess I'm a fool for following up my own posts since some jackass named 'child-duckling' marked those an answers? Hah, after 4 days I guess there's a fat chance of getting any of those around here........

**Comment by dabl2** [comment]
Just found [this thread](https://github.com/cryptofyre/Apple-Music-Electron/discussions/107).

Tried  "disableCORS": true in config.json, killed the app from the tray and restarted and saw no change in behavior.

Sounds like I need the beta to try the authMode setting, will try that next.

**Reply by dabl2** [reply]
Looks like the v2.4.0 release does not have the authMode setting and I'm not seeing any .exe for any of the 2.4.0 betas

Also noticed the v2.4.0 release didn't create a folder in Documents and/or a config.json file

**Reply by ghost** [reply]
Exactly! I have been looking for it but it just doesnt work, can't get past the login itself. Don't see any settings.

**Reply by coredev-uk** [reply]
2.4.0 uses a GUI which can be accessed through `CTRL + ALT + S` or by clicking on the top right (your profile picture) and then clicking 'App Settings', this should open the GUI. On the next beta release, themes will be re-added back to the appdata folder for users who want to create their own. But any configuration can be done in the GUI.

**Comment by cryptofyre** [comment]
Settings on the 2.4.0 build have been moved into its own GUI. Access this by pressing ```CTRL + ALT + S``` and to fix the login issue enable "Authentication Mode" then restart the app.

**Reply by cryptofyre** [reply]
I recommend running version 2.4.0 if you continue running into issues with 2.3.0

**Reply by ghost** [reply]
Thank god!!
It is working now. :)

I enabled Authentication mode and restarted. Baam, its working now. Cheers.

**Comment by dabl2** [comment]
Cool thanks, yeah once I used CTRL + ALT + S, enabled authentication and restarted my login was successful.

Re-reading the text on the releases page "Settings Menu can be invoked with CTRL+ALT+S or by pressing the "App Settings" button by pressing the User Picture icon and finding it" clearly CTRL+ALT+S is referenced so I'm not sure why I didn't try it however I was focusing on the text attempting to describe how to get to the 'App Settings button' which of course does not exist on first time startup before logging in (which was the problem).  So, probably best to say something like 'On first time startup use CTRL+ALT+S to get to the Settings Menu" or better simply make a button for the Settings Menu that is (also) available before the first time login and not make it so hidden etc.
